docs/shared/recipes/add-stack/add-lit.md

```diff
@@ -0,0 +1,206 @@
+# Add a New Lit Project
+
+The code for this example is available on Github:
+
+{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/lit" /%}
+
+**Supported Features**
+
+Because we are not using an Nx plugin for Lit, there are few items we'll have to configure manually. We'll have to configure our own build system. There are no pre-created Lit-specific code generators. And we'll have to take care of updating any framework dependencies as needed.
+
+{% pill url="/core-features/run-tasks" %}✅ Run Tasks{% /pill %}
```

Comment:
in the last call we had (while u were on vacation) we briefly saw this and weren't sure how useful it is...kinda the concern I had previously. I understand the attempt to re-surface some of these features but I'm wondering about the maintainability implications.

Like most of the time all of them would be ticked apart from the "Use Task Executors" and "Use Code Generators" and "Automtae Updating Framework" probably...basically whenever it is not generated by either one of our plugins or a community plugin.

Reply:
Yes, a lot of them will be similar, but people aren't going to be reading all of the recipes back to back.  They only care about their particular stack.  Having a lot of check marks is a good thing so we can show that most of the Nx functionality works.  We do need to be clear about which parts of Nx will not be available for their tech stack though.

The whole goal of this effort is to make it clear to people that even though we don't have an official plugin for a stack, they can get a lot of value out of using Nx with it.  That's what these checkmarks emphasize.